install/README.md

```diff
@@ -0,0 +1,22 @@
+# Script for complete Pravega Installation
+
+The purpose of this script is to sequentially install all the dependencies (i.e. Operators, Zookeeper and Bookkeeper) and Pravega Components in the right order.
+
+## Prerequisites
+
+  - Kubernetes 1.10+ with Beta APIs
+  - Helm 2.10+
+  - [Tier 2 Setup](https://github.com/pravega/pravega-operator#set-up-tier-2-storage)
+  - Copy the necessary charts to the right location
+
+To copy the charts from [Zookeeper Operator](https://github.com/pravega/zookeeper-operator/tree/master/charts) and [Bookkeeper Operator](https://github.com/pravega/bookkeeper-operator/tree/master/charts) repositories inside the charts directory of this repository.
+
+Create separate sub-directories for [zookeeper-operator](https://github.com/pravega/zookeeper-operator/tree/master/charts/zookeeper-operator), [zookeeper](https://github.com/pravega/zookeeper-operator/tree/master/charts/zookeeper), [bookkeeper-operator](https://github.com/pravega/bookkeeper-operator/tree/master/charts/pravega-operator) and [bookkeeper](https://github.com/pravega/bookkeeper-operator/tree/master/charts/pravega) charts alongside the directories for [pravega-operator](https://github.com/pravega/pravega-operator/tree/master/charts/pravega-operator) and [pravega](https://github.com/pravega/pravega-operator/tree/master/charts/pravega) charts inside the [charts](https://github.com/pravega/pravega-operator/tree/master/charts) directory.
+
+## Installing the Pravega Cluster
```

Comment:
Script should take install/delete as argument and based on value passed should install or delete the pravega cluster. 
Please rename the parent folder for the script since this script will also be used to cleanup a pravega cluster.